projects/ngx-oxford/src/lib/oxford.pipe.ts

```diff
@@ -1,5 +1,10 @@
 import { Pipe, PipeTransform } from '@angular/core';
 
+interface TruncateOptions {
```

Comment:
Could you rename this to OxfordOptions or something more general? I think this would turn into general options (not just for truncation) if any more were added.

Reply:
Done